localization/localization_error_monitor/src/node.cpp

```diff
@@ -40,7 +41,7 @@ LocalizationErrorMonitor::LocalizationErrorMonitor()
   error_ellipse_size_lateral_direction_ =
     this->declare_parameter("error_ellipse_size_lateral_direction", 0.3);
   warn_ellipse_size_lateral_direction_ =
-    this->declare_parameter("warn_ellipse_size_lateral_direction", 0.2);
+    this->declare_parameter("warn_ellipse_size_lateral_direction", 0.25);
```

Comment:
Same as [this](https://github.com/autowarefoundation/autoware.universe/pull/4964/files#diff-9ea4b7dd951014d6499494a7e340fca96ec106bc747f525e655f6acbe94d8954R7).